ui/app/src/state/epics/content.ts

```diff
@@ -508,8 +510,21 @@ const content: CrafterCMSEpic[] = [
           );
         }
       })
-    )
+    ),
   // endregion
+  // region Emit System Event
+  (action$, state$) =>
+    action$.pipe(
+      ofType(emitSystemEvent.type),
+      withLatestFrom(state$),
+      filter(([{ payload }]) => payload.type === lockContentEvent.type),
```

Comment:
Lets go ahead and dispatch system events to the store (on epics/system) so we don't have to do these filterings like we've had to do here and in the content reducer. On the emitSystemEvent epic, after the tap, map(action => action) and change here and on the reducer to simply wait for the actual action rather than having to inspect the system events.